File: Problem018/Solution.cpp
#include <vector>
#include <string>
using namespace std;

class Solution
{
public:
    vector<int> findWordsContaining(vector<string> &words, char x)
    {
        vector<int> result;
        int len = words.size();
        for (int i = 0; i < len; ++i)
        {
            string s = words[i];
            if (s.find(x) != string::npos)
            {
                result.push_back(i);
            }
        }
        return result;
    }
};

File: Problem018/Solution.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        List<Integer> result = new ArrayList<>();
        int len = words.length;
        for (int i = 0; i < len; ++i) {
            String s = words[i];
            boolean is_found = s.contains(s.valueOf(x));
            if (is_found)
                result.add(i);
        }
        return result;
    }
}